### PR TITLE
Add walkthroughs to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by zadjii-msft**
Change the "good first issues" text to talk about walkthroughs instead.

### CODE REVIEW

**Comment by zadjii-msft** [comment]
Both, both is good

**Comment by tusharsnx** [comment]
Off-topic, did you just read my comment on the latest terminal showcase on yt 😅  ?

image

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ If you don't have any additional info/context to add but would like to indicate
...
 
-The best place to start is the list of ["good first issue"](https://github.com/microsoft/terminal/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22++label%3A%22good+first+issue%22+)s. These are bugs or tasks that we on the team believe would be easier to implement for someone without any prior experience in the codebase. Once you're feeling more comfortable in the codebase, feel free to just use the ["Help Wanted"](https://github.com/microsoft/terminal/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22+) label, or just find an issue you're interested in and hop in!
+The best place to start is the list of ["walkthroughs"](https://aka.ms/terminal-walkthroughs)s. This is a collection of issues where we've written a "walkthrough", a little guides to help get started with a particular issue. These are usually good first issues, and are a great way to get familiar with the codebase. Once you're feeling more comfortable in the codebase, feel free to just use the ["Help Wanted"](https://github.com/microsoft/terminal/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22+) label, or just find any issue you're interested in and hop in!
```
Comment (DHowett) [inline CONTRIBUTING.md:104]: This says `"walkthroughs"s`, FWIW

**Review by DHowett — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (DHowett) [reply CONTRIBUTING.md:104]: it also says "a little guide**s** to help get started"

**Comment by zadjii-msft** [comment]
> Off-topic, did you just read my comment on the latest terminal showcase on yt 😅 ?

lol I didn't know that had already been posted! I thought that was supposed to be going live _this_ week 🤦 

I legit was writing a blog post to go with that video this morning and Copilot was all
image

to which I said "damn that's a good idea"

**Review by DHowett — APPROVED** [review]

**Pull request merged**